src/main/java/core/basesyntax/strategy/impl/BalanceHandlerImpl.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceHandlerImpl implements OperationHandler {

```

Comment:
```suggestion
public class BalanceHandler implements OperationHandler {
```
the same for other implementations